vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -232,18 +232,18 @@ def process_weights_after_loading(self, layer: Module) -> None:
             weight = layer.weight
             layer.weight = Parameter(weight.t(), requires_grad=False)
 
-            # ACT_SCALE
+            # INPUT_ACTIVATION_SCALE
```

Comment:
Might be worth calling this `INPUT ACTIVATION SCALE` to be consistent with the comment above :)